src/ontology/cl-edit.owl

```diff
@@ -6673,11 +6674,11 @@ SubClassOf(obo:CL_0000386 obo:CL_0000630)
 
 # Class: obo:CL_0000387 (hemocyte (sensu Arthropoda))
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/B978-012369493-5.50008-0") obo:IAO_0000115 obo:CL_0000387 "A blood cell of the circulatory system of arthropods.")
```

Comment:
dbxref behind a paywall. Do not see this cell type in the "Publisher Summary".

Reply:
Please clarify. Is that a request to replace the cross-reference by a publicly available reference, because there is a policy that all cross-references should be usable by everybody (without requiring to go through a paywall)? Or are you merely saying that, for lack of public access, you didn’t check yourself the correctness of the definition but you are happy to “defer to [my] subject matter knowledge”?

Reply:
Since the reference is behind a paywall, I could not cross check that it provides evidence to support the content.
I don't think there is a policy / requirement to use a public resource, though if it is not public, a reviewer without access cannot check if it aligns with the content of the annotation.

Reply:
Duly noted. I’ll try to prioritise publicly available references in future PRs, or at least to have additional references that are publicly available.